fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -120,6 +120,10 @@ def self.is_mac?
       self.mac?
     end
 
+    def self.is_windows?
+      self.windows?
```

Comment:
you can also use `alias_method`

Reply:
Oh, great idea, and we could use the same for other methods also, like `is_mac?`, etc., right? :)